telemetry/definitions/commonDefinitions.json

```diff
@@ -3294,6 +3438,419 @@
+            "description": "The code transform button in the devtools to initiate the job but there is a validation error.",
+            "metadata": [
+                {
+                    "type": "codeTransformPreValidationError",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                { "type": "result", "required": true },
+                { "type": "reason", "required": true }
+            ]
+        },
+        {
+            "name": "codeTransform_jobIsStartedFromUserPopupClick",
+            "description": "The user initiates a transform job from the pre transform user modal window.",
+            "metadata": [
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_jobIsResumedAfterIdeClose",
+            "description": "The user was previously running a transformation and came back to IDE to see it still running",
+            "metadata": [
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformStatus",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_jobIsClosedDuringIdeRun",
+            "description": "The user closes their IDE and stops viewing the transformation.",
+            "metadata": [
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformStatus",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_jobIsStartedFromChatPrompt",
+            "description": "The user initiates a transform job from Amazon Q chat prompt.",
+            "metadata": [
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_jobIsCanceledFromUserPopupClick",
+            "description": "The code transform button to initiate the pre transform user modal.",
+            "metadata": [
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_jobStartedCompleteFromPopupDialog",
```

Comment:
This name was in question for the VSCode PR -> https://github.com/aws/aws-toolkit-vscode/pull/4131/files

We already shipped this and are graphing against this name for Intellij and our Kibana dashboards. If we change this name, we will have to backfill the name change and that doesn't seem worth the effort.

<img width="785" alt="Screenshot 2023-12-05 at 9 53 34 AM" src="https://github.com/aws/aws-toolkit-common/assets/38665452/9ec85967-2ec5-4cff-8e62-cc70879b5508">

Reply:
This is passive because its fired AFTER successful completion of steps. A final funnel metric.

Reply:
got it